### ISSUE TITLE
gsts cant find playwright chromium engine

### ISSUE THREAD

**Issue by p-hash** [ruimarinho/gsts#96]
gsts tries to get chromium-1045 and does not find it, but chromium-1071 is already installed.

```
$ gsts --username ... --aws-role-arn ... --sp-id ... --idp-id ... --json -v
ℹ Attempting to retrieve session expiration credentials
ℹ Credentials file does not exist at /home/paul/.aws/gsts_credentials_stagin
browserType.launchPersistentContext: Executable doesn't exist at /home/paul/.cache/ms-playwright/chromium-1045/chrome-linux/chrome
╔═════════════════════════════════════════════════════════════════════════╗
║ Looks like Playwright Test or Playwright was just installed or updated. ║
║ Please run the following command to download new browsers:              ║
║                                                                         ║
║     npx playwright install                                              ║
║                                                                         ║
║ <3 Playwright Team                                                      ║
╚═════════════════════════════════════════════════════════════════════════╝
    at async /usr/local/share/.config/yarn/global/node_modules/gsts/index.js:226:19 {
  name: 'Error'
}
$ ls -1 ~/.cache/ms-playwright/
chromium-1071
ffmpeg-1009
firefox-1419
webkit-1869
```

**Comment by p-hash** [comment]
Solved by installing specific version of playwright
` npm install playwright@v1.30.0`

**Comment by p-hash** [comment]
Also you can specify correct `--engine-executable-path` , for example `/home/user/.cache/ms-playwright/chromium-1071/chrome-linux/chrome`

**Comment by ruimarinho** [comment]
@p-hash is this a new install from scratch? How did Chrome get upgraded?

**Comment by p-hash** [comment]
It is a new install, yes

**Comment by ruimarinho** [comment]
Thanks for the report @p-hash! Should be fixed in v5.0.1. Could you please give it a try?
